### PR TITLE
Too many snapshots

### PR DESCRIPTION
**Pull request by utkarshmani1997**
This commit increase the maximumChainLength default value and also adds capability 
to configure it via environment variable.

### CODE REVIEW

File: app/replica.go [hunk 1]
```diff
@@ -144,6 +147,13 @@ func startReplica(c *cli.Context) error {
 		return errors.New("directory name is required")
 	}
 
+	types.MaxChainLength, _ = strconv.Atoi(os.Getenv("MAX_CHAIN_LENGTH"))
+	if types.MaxChainLength == 0 {
+		logrus.Infof("MAX_CHAIN_LENGTH env not set, default value is 250")
```
Comment (kmova) [inline app/replica.go]: 255?

Comment (utkarshmani1997) [reply app/replica.go]: it's 250 @kmova

File: replica/replica.go [hunk 2]
```diff
@@ -33,7 +33,7 @@ const (
 	diskPrefix         = "volume-snap-"
 	diskSuffix         = ".img"
 	diskName           = diskPrefix + "%s" + diskSuffix
-	maximumChainLength = 250
+	maximumChainLength = 300
```
Comment (kmova) [inline replica/replica.go]: Can we make this default as 512?

**Comment by vishnuitta** [comment]
@utkarshmani1997 please cherry-pick this to v1.0.x and v0.9.x as well

**Pull request merged**